src/eval.rs

```diff
@@ -3922,7 +3932,7 @@ mod test {
         let expr2 = "(% 100u64 3u64)";
         let res2 = s.uint64(1);
 
-        let expr3 = "(/ 100u64 0u64)";
```

Comment:
This will also need to be changed in `nova.rs` (`outer_prove_test_u64_mod`). Making that pass will likely also require a circuit change.

Reply:
I modified the circuit to add this error recently: https://github.com/lurk-lang/lurk-rs/blob/master/src/circuit/circuit_frame.rs#L3971

Reply:
It is tested here: https://github.com/lurk-lang/lurk-rs/blob/master/src/proof/nova.rs#L2945

Reply:
Ah, good. It's still probably worth changing for consistency and since there's no need to test `/` in the mod test (unless I'm missing something). It looks like this was just a copy-paste error in the first version of the tests.